src/System.Drawing.Common/src/misc/ClientUtils.cs

```diff
@@ -124,15 +124,15 @@ public static bool IsEnumValid_NotSequential(System.Enum enumValue, int value, p
 
 #if DEBUG      
         [ThreadStatic]
-        private static Hashtable t_enumValueInfo;
-        public const int MAXCACHE = 300;  // we think we're going to get O(100) of these, put in a tripwire if it gets larger.
+        private static Hashtable s_enumValueInfo;
```

Comment:
I'll submit a PR. Meanwhile I assume this comment doesn't apply to System.Drawing.Common :)